patches/icon.patch

```diff
@@ -0,0 +1,246 @@
+diff --git a/uppsrc/ide/icon.svg b/uppsrc/ide/icon.svg
```

Comment:
Already  in the upstream https://github.com/ultimatepp/ultimatepp/commit/123f8e0354e834101b094f486f7a53c465cc4d7c